frontend/core-ui/src/modules/products/detail/utils/blockNode.ts

```
@@ -0,0 +1,125 @@
+          }
+          // Add more block types as needed
+          return "";
+        })
+        .filter(Boolean)
+        .join("");
+    } catch (e) {
+      console.error("Error converting BlockNote to HTML:", e);
+      return "";
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)